server/controllers/courseCtrl.js

```diff
@@ -342,7 +342,11 @@ function _parseRounds(roundInfos, courseCode, language, webContext) {
   const courseRoundList = {}
   for (const roundInfo of roundInfos) {
     courseRound = _getRound(roundInfo, language)
-    const { round_course_term: yearAndTermArr, roundId: ladokRoundId } = courseRound
+    const {
+      round_course_term: yearAndTermArr,
+      roundId: ladokRoundId,
```

Comment:
`roundId: ladokRoundId` has to be removed

Reply:
Removed